libraries/timeslider/src/locale/inv.json

```diff
@@ -0,0 +1,39 @@
+{
+    "language-web-incubator-time-slider-title": "Time Slider",
+    "language-designer-time-slider-slider-min-description": "The minimum time for the complete time extent represented by this time slider.",
```

Comment:
Seems a lot of this stuff is unused now and can be scrapped?

Reply:
Scrapped most language strings.